src/pages/Home.js

```diff
@@ -181,124 +187,177 @@ function Home() {
   };
   // Pagination
 
-  let Foodspots = () => {
-    return currentRecords.map((item, index) => {
-      return (
-        <Link to={`/foodspot/` + item.id} state={item.id} key={item.id}>
-          <li key={index}>{item.title}</li>
-        </Link>
-      );
-    });
-  };
-
   return (
-    <div className="flex justify-center my-[30px]">
-      <div className="filter-section">
-        <div className="filter-buttons">
+    <div className="contentWrapper max-w-[1915px] mx-auto my-[100px] flex px-[100px]">
+      <div className="filterSection">
```

Comment:
Die ganze Filter Section sollte in eine eigene Komponente ausgelagert werden, damit die "Home.js" übersichtlich bleibt und es eine klare Trennung gibt.